project/FrontendBuild.scala

```diff
@@ -18,7 +18,8 @@ object FrontendBuild extends Build with MicroService {
     "com.github.kxbmap" %% "configs" % "0.4.4",
     "com.eclipsesource" %% "play-json-schema-validator" % "0.8.9",
     "uk.gov.hmrc" %% "play-whitelist-filter" % "2.0.0",
-    "uk.gov.hmrc" %% "zap-automation" % "0.15.0"
```

Comment:
Why has this been removed?

Reply:
it's already being brought in as a test dependency below - I didn't think we needed it outside of testing